src/config.js

```diff
@@ -12,6 +12,14 @@ function setConfigFileFolder(folder) {
   config = config
     .file('env-file', environmentConfigFile) // environment file is always of higher priority.
     .file('common-file', commonConfigFile);
+
+  putValuesIntoProcessEnv(config.get());
+}
+
+function putValuesIntoProcessEnv(config) {
+  for (let key in config) {
+    if (typeof config[key] === 'string' && !process.env[key]) process.env[key] = config[key];
```

Comment:
I prefer to have curly braces in single line if as well for better readability.

```
  if (typeof config[key] === 'string' && !process.env[key]) {
     process.env[key] = config[key];
  }
```

If you can, please add it to eslint definition as well.